Empire/CampaignAsset.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Empire;
+
+class Campaign {
```

Comment:
The PHP autoloader needs this class to be in its own file.

Reply:
Thank you, Ryan! Yesterday we spend a couple of hours investigating why this file isn't loaded. 🙂